houston/workspace_teams.go

```diff
@@ -0,0 +1,76 @@
+package houston
+
+// AddTeamToWorkspace - add a team to a workspace
+func (h ClientImplementation) AddWorkspaceTeam(workspaceUUID, teamUuid, role string) (*Workspace, error) {
```

Comment:
as in all other Houston methods
*UUID -> *ID

workspaceUUID -> workspaceID

teamUuid -> teamID